experimental/examples/chat-mpp/shared/src/commonMain/kotlin/ChatApp.kt

```diff
@@ -12,34 +15,33 @@ val friendMessages = listOf(
     "Nice to see you!",
     "Multiline\ntext\nmessage"
 )
+val store = CoroutineScope(SupervisorJob()).createStore()
 
 @Composable
-internal fun ChatApp() {
-    val coroutineScope = rememberCoroutineScope()
-    val store = remember { coroutineScope.createStore() }
+internal fun ChatApp(displayTextField:Boolean = true) {
     val state by store.stateFlow.collectAsState()
-
-    MaterialTheme {
-        Box(modifier = Modifier.fillMaxSize()) {
-            Scaffold(
```

Comment:
I would  leave Scaffold for non iOS targets.

Reply:
Done